rueidisprob/slidingbloomfilter.go

```diff
@@ -167,10 +167,6 @@ redis.call('SET', nextCounterKey, 0)
 	lastRotationSuffix = ":lr"
 )
 
-type Logger interface {
```

Comment:
@rueian we miss this one on the latest PR, some garbage from the older implementation of the slidingwindow